packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/apis/update.test.ts

```diff
@@ -149,15 +149,15 @@ describe('#update', () => {
       it(`should use the ES get action then index action when type is not multi-namespace for existing objects`, async () => {
         const type = 'index-pattern';
         const id = 'logstash-*';
-        migrator.migrateDocument.mockImplementationOnce((doc) => ({ ...doc, migrated: true }));
+        migrator.migrateDocument.mockImplementationOnce((doc) => ({ ...doc }));
```

Comment:
Tests were adapted due to the validation being enabled for all types (see below)